python/scannerpy/database.py

```diff
@@ -323,6 +325,18 @@ def start_cluster(self, master, workers):
             assert res
             res = self._connect_to_master()
             assert res
+
+            # Start up heartbeat to keep master alive
+            def heartbeat_task():
```

Comment:
This isn't necessary in debug mode, right? Since the master/worker already get cleaned up with the Python process?